apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -262,15 +299,20 @@ export default NextAuth({
             name: true,
             email: true,
             role: true,
+            metadata: true,
           },
         });
 
-        if (!existingUser) {
+        if (!existingUserRaw) {
           return token;
         }
 
+        const { metadata, ...existingUser } = existingUserRaw;
+        const parsedMetadata = userMetadata.parse(metadata);
+
         return {
           ...existingUser,
+          sessionTimeout: parsedMetadata?.sessionTimeout,
```

Comment:
Making sure we have the `sessionTimeout` value to use it when encoding the session token